hal/qspi_api.h

```diff
@@ -49,7 +50,6 @@ typedef enum qspi_bus_width {
 /** Address size
  */
 typedef enum qspi_address_size {
-    QSPI_CFG_ADDR_SIZE_NONE,
```

Comment:
Can I know why we are removing QSPI_CFG_ADDR_SIZE_NONE/QSPI_CFG_ALT_SIZE_NONE. Theoretically they can be none.

Reply:
The description or commit should contain the reason:

> To be able to skip any phase in the format, use bus width set to NONE. Previously this
was done via SIZE definitions, this however did not provide flexibility to skip instruction phase
or data (as all of them can be done via 0-4 lines).

Reply:
I am open to suggestions, I find current size none limiting, this is the property that each phase has. We can add boolean enabled for instance instead, or any other?